.travis.yml

```diff
@@ -2,7 +2,6 @@ language: ruby
 sudo: false
 cache: bundler
 rvm:
-  - 2.1.10
   - 2.2.7
   - 2.3.4
   - 2.4.1
```

Comment:
Can you bump these all by one to get the latest: 2.2.8, 2.3.5, 2.4.2

Reply:
Done.